content/blog/2024-06-20-why-rse.md

```diff
@@ -0,0 +1,100 @@
+
+**Long answer**
+
+RSE can leverage all the above issues and more by accompanying researchers in their software development journey. RSEs' technical knowledge and creativity on solving problems allow seeing where researchers would benefit the most. Here are a few examples/use cases: 
+
+* Researchers have great ideas, but might not know how, or have time to develop the corresponding tools 
+* Help to get rid of repetitive or time consuming steps (that users don't want to do themselves) by introducing automatization 
+* Write documentation so the information doesn't get lost when people leave
+* Make the regular technology updates to prevent that it become obsolete and break
+* Supporting when switching from laptop to compute clusters 
+* Many groups need the same thing, RSEs know how to create flexible templates to simplify e.g. spinning up a website to present research outputs
+* … what other good examples? 
+
+They can also act as intermediate in technical discussions about computing needs, as they know the needs of researchers since they are involved in research in a daily basis, and are also technology-aware people. 
+
+As a summary, researchers with very deep expertise and RSEs with shallower but wider range of interests and skills complement each other very efficiently. 
+
+#### How organisations can help?
+
+Many organisations already have people doin RSE work, but the people and the organisation might not be aware of the term RSE. To give those people a sense of belonging and knowing that there is others like them, organisations could:
+
+1) Support community involvement and self development
+    - Give people room to be active in RSE communities, attend conferences and trainings beyond academic work.
+
+2) Spread the word - be an RSE ambassador
+    - Recognize and highlight RSE (often behind the scenes and non-academic) work
+    - Invite RSEs to talk about their work
+    - Support RSEs building their own network within the organisation (connect them to administration, other RSEs, researchers, etc)
+
+3) Make it a THING (administrative existance)
+    - Hire RSEs
+    - If you have them already, gather them ("virtual team" or central RSE group)
+    - Offer support to spread good enough software practices in research groups
+    - Train the trainer events are important and needed 
+    - Support formation of RSE groups by providing guidance about practices for forming a new entity in the organisation (group, service, etc) and finding the right integration points
+    - Share organizational knowledge on how get internal funding for RSE-type work
+    
+4) Funding
+    - Inlcude RSEs at the stage of grant proposal writing
+    - Provide central RSE funding to support smaller projects and internal tool developments 
+    - RSE benefits from open ended, non temporary funding - like other techical supports
+    - training and collaboration funding (e.g. EuroCC, EOSC)
+
```

Comment:
5) acknowledge them.
While the contribution of the RSE team/engineer might not be enough for a co-author ship, acknowledge the role for RSE in your analysis in the acknowledgment of the article. At the digital lab we actually provide two ready statements for this:

For coding/data help:
"Code development and data management [DELETE AS NECESSARY] were supported by the Industrial Ecology Digital Laboratory."

For access to the servers:
"The computations/simulations/[SIMILAR] were performed on resources provided by the Industrial Ecology Digital Laboratory."

Reply:
Oh yes! This is indeed missing, and a step that does not "cost" organizations anything. 
Should this actually be number 1 or 2 rather than 5?

Reply:
Excellent suggestion! I would put it at the same level as 2)

Reply:
added a suggestion to the text, pleas edit as you see fit